plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -86,6 +86,14 @@ public boolean isDeprecated() {
         return deprecated;
     }
 
+    public boolean isIgnoredOnExport() {
+        return isDeprecated() || isRestricted() || isBlackListedForExport();
+    }
+
+    private boolean isBlackListedForExport() {
+        return toString().equals("defaultProperties(class: class hudson.tools.ToolProperty, multiple: true)");
```

Comment:
We discussed having a resource file where we could blacklist, pretty much like [JEP-200](https://jenkins.io/blog/2018/01/13/jep-200/)

Reply:
Yes, it would be my preference as well. We could basically copy the implementation from there

Reply:
my intention as well, didn't have time to implement it on the train.
Do we need plugins to be able to contribute to it or just maintain the master file in jcasc for now?

Reply:
I would prefer plugins were able to contribute to it in their plugins much like JEP-200 😄

Reply:
I would prefer plugins were able to contribute to it in their plugins much like JEP-200 😄 

perhaps calling toString is not the best thing?
I would prefer we used the fully-qualified symbol name `hudson.tools.ToolProperty#defaultProperties`

Reply:
I don't think the full information is available there at this time but can double check / look at making it available

Reply:
Fully qualified name for JCasC may be a tricky thing, but it would be great.
Note that we also need to handle inheritance and other similar funny things

Reply:
`defaultProperties(class: class hudson.tools.ToolProperty, multiple: true)` this is fine too but I do not know if this is how all methods will look like 😓

Reply:
I'll try write a test that blacklists a few methods to show examples

Reply:
Found the jep-200 code... linking here, as it took awhile to track down:
https://github.com/jenkinsci/jenkins/pull/3120/files#diff-c053f4fb731f9a114ac561e1fd688413R154

Reply:
I've gone with a simpler approach for now, which can be iterated on.

Reply:
Class name for the name isn't available in attribute currently, I've put examples for all attributes in the test I ran in the main PR comments